api/CloseEvent.json

```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "CloseEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CloseEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "8",
+            "notes": "Prior to Firefox 8, the WebSocket close event was sent to the listener as a simple event. Prior to Firefox 12, the close code CLOSE_NORMAL was used when the channel was closed due to an unexpected error or unspecified error conditoin."
+          },
+          "firefox_android": {
+            "version_added": "8"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "initCloseEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CloseEvent/initCloseEvent",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": null
```

Comment:
Ditto all 3. See above.

Reply:
@jpmedley According to the [Chromium Code Search](https://cs.chromium.org/search/?q=initCloseEvent&ssfr=1&type=cs), `initCloseEvent` isn't actually supported by Chrome. Can you elaborate on that?

Reply:
For the time being, I have set all to `false` in c2d8949f0be1c60887c66b19230011ffdac21ffa.